packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.js

```diff
@@ -166,7 +166,6 @@ describe('ReactHooksWithNoopRenderer', () => {
     // Schedule some updates
     if (gate(flags => flags.enableSyncDefaultUpdates)) {
       React.startTransition(() => {
-        // TODO: Batched updates need to be inside startTransition?
```

Comment:
The transition is just to make it interruptible below, not related to batched updates.

Reply:
Actually yeah, batched updates shouldn't be needed here in either branch, but without it we fail because the update is not wrapped in `act`, so I assume this is a hack to get around that warning?

Reply:
What you could do instead is wrap the whole block in `act`.

Reply:
So the batchedUpdates here isn't important?

Reply:
Nah it was added for the reason you suggested, to avoid the `act` warning: https://github.com/facebook/react/commit/d278a3ff8b2863afb9ab7807e42d31f33c84eb78